microsoft/PowerToys
# [ARM64] Common Projects configuration cleanup

## Summary of the Pull Request

**What is this about:**
This PR updates the common projects under `src\common` with the necessary configuration to support being built as ARM64, such as removing x64 explicit configuration and adding ARM64 platform.

**What is included in the PR:** 
PlatformTarget removed and RuntimeIdentifiers added, since we're targeting .NET 6.
Added ARM64 to Platforms in csprojs
All projects under src\common will now build under ARM64 when solution is targeting.

**How does someone test / validate:** 
Set solution build config to Release|ARM64, run a rebuild on `PowerToys.Interop` and `ManagedCommon`.
These should build successfully

## Quality Checklist

- [X] **Linked issue:** #490

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
